static/src/javascripts/projects/common/utils/ajax-promise.js

```diff
@@ -13,6 +15,11 @@ define([
             })
             .fail(function (request, text, err) {
                 var error = err ? err : new Error(text);
+
+                var statusText = (err && err.statusText) || '';
+                var statusCode = (err && err.status) || '';
+                raven.captureException('Error retrieving data (' + text + ') (Status: ' + statusCode + ') (StatusText: ' + statusText + ')');
```

Comment:
BTW: The `when` Promise library supports a global event handler for unhandled promise rejections, so I have started migrating us to that.